utils/CMakeLists.txt

```diff
@@ -23,11 +23,11 @@ add_custom_target(OMONNXOpsIncTranslation
         DEPENDS OMONNXOpsTableGenIncGen
                 OMONNXOpsBuildTableIncGen)
 
-# Invoke gen_doc.py to obtain ONNXOps.td.inc, OpBuildTable.inc.
+# Invoke gen_onnx_mlir.py to obtain ONNXOps.td.inc, OpBuildTable.inc.
```

Comment:
MLONNXOps.td.inc? MLOpBuildTable.inc?

Reply:
I will fix these minor errors in my next patch.